src/main/java/frc/team166/robot/RobotMap.java

```diff
@@ -20,6 +20,8 @@
         public final static int frontleft = 3;
```

Comment:
I think these are left over from the practice bot? Let's remove them until the Drive subsystem is actually implemented.